diff -urN xemacs-21.5.27.org/configure xemacs-21.5.27/configure
--- xemacs-21.5.27.org/configure	2006-05-16 06:16:04.000000000 +0200
+++ xemacs-21.5.27/configure	2007-01-04 03:01:43.566429500 +0100
@@ -39268,7 +39268,7 @@
 T=""
 for W in $ac_configure_args; do if test -z "$T"; then T="$W"; else T="$T $W"; fi; done
 ac_configure_args="$T"
-
+ac_configure_args=" $ac_configure_args"
 cat >>confdefs.h <<_ACEOF
 #define EMACS_CONFIGURATION "$ac_cv_build"
 _ACEOF
diff -urN xemacs-21.5.27.org/configure.ac xemacs-21.5.27/configure.ac
--- xemacs-21.5.27.org/configure.ac	2006-04-23 18:11:17.000000000 +0200
+++ xemacs-21.5.27/configure.ac	2007-01-04 03:01:04.311976250 +0100
@@ -22,8 +22,6 @@
 dnl Due to a dependence on the implementation of certain internal autoconf
 dnl macros, die if any version other than 2.59 is used.
 AC_PREREQ(2.59)dnl
-m4_if(m4_version_compare(m4_defn([m4_PACKAGE_VERSION]), [2.59]), 1,
-	[m4_fatal([This script can only be generated with autoconf 2.59])], [])dnl
 AC_INIT([XEmacs],[21.5],[xemacs-beta@xemacs.org])dnl
 AC_CONFIG_SRCDIR(src/lisp.h)dnl
 dnl
@@ -5844,6 +5842,9 @@
 fi
 
 XE_SPACE(ac_configure_args, $ac_configure_args)
+dnl Put back the leading space to work around problems with
+dnl 'config.status --recheck' under autoconf 2.60 and 2.61.
+ac_configure_args=" $ac_configure_args"
 AC_DEFINE_UNQUOTED(EMACS_CONFIGURATION, "$ac_cv_build")
 AC_DEFINE_UNQUOTED(EMACS_CONFIG_OPTIONS, "$ac_configure_args")
 
