diff -urN xemacs-21.4.2.org/configure.in xemacs-21.4.2/configure.in
--- xemacs-21.4.2.org/configure.in	Fri May 18 01:06:20 2001
+++ xemacs-21.4.2/configure.in	Fri May 18 01:06:35 2001
@@ -339,7 +339,7 @@
 pkgdir='${datadir}/${instvardir}/lisp'
 package_path=''
 etcdir='${datadir}/${instvardir}/etc'
-archlibdir='${datadir}/${instvardir}/${configuration}'
+archlibdir='${libdir}/${instvardir}/${configuration}'
 docdir='${archlibdir}'
 with_netinstall="no"
 with_prefix='yes'
