diff -urN xemacs-21.4.3.org/src/event-gtk.c xemacs-21.4.3/src/event-gtk.c
--- xemacs-21.4.3.org/src/event-gtk.c	Mon Jun  4 05:52:34 2001
+++ xemacs-21.4.3/src/event-gtk.c	Mon Jun  4 05:53:54 2001
@@ -1253,6 +1253,26 @@
 	if (*state & gd->HyperMask)  modifiers |= XEMACS_MOD_HYPER;
 	if (*state & gd->AltMask)    modifiers |= XEMACS_MOD_ALT;
 
+	{
+	  int numero_de_botao = -1;
+
+	  if (!key_event_p)
+	    numero_de_botao = gdk_event->button.button;
+
+	  /* the button gets noted either in the button or the modifiers
+	     field, but not both. */
+	  if (numero_de_botao != 1 && (*state & GDK_BUTTON1_MASK))
+	    modifiers |= XEMACS_MOD_BUTTON1;
+	  if (numero_de_botao != 2 && (*state & GDK_BUTTON2_MASK))
+	    modifiers |= XEMACS_MOD_BUTTON2;
+	  if (numero_de_botao != 3 && (*state & GDK_BUTTON3_MASK))
+	    modifiers |= XEMACS_MOD_BUTTON3;
+	  if (numero_de_botao != 4 && (*state & GDK_BUTTON4_MASK))
+	    modifiers |= XEMACS_MOD_BUTTON4;
+	  if (numero_de_botao != 5 && (*state & GDK_BUTTON5_MASK))
+	    modifiers |= XEMACS_MOD_BUTTON5;
+	}	
+
 	/* Ignore the Caps_Lock key if:
 	   - any other modifiers are down, so that Caps_Lock doesn't
 	   turn C-x into C-X, which would suck.
@@ -1374,6 +1394,12 @@
         if (mask & gd->SuperMask)	modifiers |= XEMACS_MOD_SUPER;
         if (mask & gd->HyperMask)	modifiers |= XEMACS_MOD_HYPER;
         if (mask & gd->AltMask)	modifiers |= XEMACS_MOD_ALT;
+	if (mask & GDK_BUTTON1_MASK)  modifiers |= XEMACS_MOD_BUTTON1;
+	if (mask & GDK_BUTTON2_MASK)  modifiers |= XEMACS_MOD_BUTTON2;
+	if (mask & GDK_BUTTON3_MASK)  modifiers |= XEMACS_MOD_BUTTON3;
+	if (mask & GDK_BUTTON4_MASK)  modifiers |= XEMACS_MOD_BUTTON4;
+	if (mask & GDK_BUTTON5_MASK)  modifiers |= XEMACS_MOD_BUTTON5;
+
         /* Currently ignores Shift_Lock but probably shouldn't
            (but it definitely should ignore Caps_Lock). */
         emacs_event->event.motion.modifiers = modifiers;
diff -urN xemacs-21.4.3.org/src/event-stream.c xemacs-21.4.3/src/event-stream.c
--- xemacs-21.4.3.org/src/event-stream.c	Mon Jun  4 05:52:34 2001
+++ xemacs-21.4.3/src/event-stream.c	Mon Jun  4 05:52:48 2001
@@ -3785,7 +3785,7 @@
 	  EQ (fun, Qscrollbar_to_right) ||
 	  EQ (fun, Qscrollbar_horizontal_drag));
 #else
-  return 0
+  return 0;
 #endif
 }
 
