diff -urN xemacs-21.4.2.org/src/event-stream.c xemacs-21.4.2/src/event-stream.c
--- xemacs-21.4.2.org/src/event-stream.c	Thu May 17 23:58:29 2001
+++ xemacs-21.4.2/src/event-stream.c	Thu May 17 23:59:37 2001
@@ -3763,6 +3763,7 @@
 static int
 is_scrollbar_event (Lisp_Object event)
 {
+#ifdef HAVE_SCROLLBARS 
   Lisp_Object fun;
 
   if (XEVENT (event)->event_type != misc_user_event)
@@ -3783,6 +3784,9 @@
 	  EQ (fun, Qscrollbar_to_left) ||
 	  EQ (fun, Qscrollbar_to_right) ||
 	  EQ (fun, Qscrollbar_horizontal_drag));
+#else
+  return 0;
+#endif
 }
 
 static void
