--- xemacs-21.5.29/configure.ac~	2009-05-18 16:51:05.000000000 +0200
+++ xemacs-21.5.29/configure.ac	2011-04-06 23:07:48.432921002 +0200
@@ -5332,6 +5332,7 @@
     AC_CHECK_HEADER(ncurses/curses.h, curses_h_file=ncurses/curses.h)
     AC_CHECK_HEADER(ncurses/term.h, term_h_file=ncurses/term.h)
     XE_ADD_OBJS(terminfo.o)
+    XE_PREPEND(-ltinfo, LIBS)
     XE_PREPEND(-lncurses, LIBS)
 
     if test "$ac_cv_header_ncurses_curses_h" != "yes" ; then
