--- xemacs/configure.in~	Wed May 12 16:29:04 1999
+++ xemacs/configure.in	Sun Aug 22 23:12:01 1999
@@ -345,7 +345,7 @@
 infopath=''
 install_pp=''
 lispdir='${datadir}/${PROGNAME}-${version}/lisp'
-dnl sitelispdir='${datadir}/xemacs/site-lisp'
+sitelispdir='${datadir}/xemacs/site-lisp'
 pkgdir='${datadir}/${PROGNAME}-${version}/lisp'
 package_path=''
 etcdir='${datadir}/${PROGNAME}-${version}/etc'
