diff -urbB xemacs-21.5.22.x/lisp/startup.el xemacs-21.5.22/lisp/startup.el
--- xemacs-21.5.22.x/lisp/startup.el	2005-05-05 18:55:41.000000000 +0200
+++ xemacs-21.5.22/lisp/startup.el	2005-10-02 23:55:19.465912250 +0200
@@ -1488,7 +1488,7 @@
 	   'external-debugging-output)
     )
 
-  (setq lisp-directory (paths-find-lisp-directory emacs-roots))
+  (setq lisp-directory "@srcdir@/lisp")
 
   (if debug-paths
       (princ (format "lisp-directory:\n%S\n" lisp-directory)
@@ -1497,8 +1497,7 @@
   (if (featurep 'mule)
       (progn
 	(setq mule-lisp-directory
-	      (paths-find-mule-lisp-directory emacs-roots
-					      lisp-directory))
+	      "@srcdir@/lisp/mule")
 	(if debug-paths
 	    (princ (format "mule-lisp-directory:\n%S\n"
 			   mule-lisp-directory)
