diff -Nru xemacs-21.1.8/src/Makefile.in.in xemacs-21.1.8.new/src/Makefile.in.in
--- xemacs-21.1.8/src/Makefile.in.in	Sat Dec 11 00:12:23 1999
+++ xemacs-21.1.8.new/src/Makefile.in.in	Sat Dec 11 00:11:51 1999
@@ -303,7 +303,7 @@
 mo_file = ${mo_dir}emacs.mo
 #endif
 
-LOADPATH =  EMACSBOOTSTRAPLOADPATH="${lispdir}:${blddir}"
+LOADPATH =  EMACSBOOTSTRAPLOADPATH="${lispdir}:${blddir}" EMACSLOADPATH="${lispdir}:${blddir}"
 DUMPENV = $(LOADPATH)
 
 release: temacs ${libsrc}DOC $(mo_file) ${other_files}
@@ -335,7 +335,7 @@
 	-$(DUMPENV) ./temacs -batch -l ${srcdir}/../lisp/loadup.el dump
 	@if test -f $@;        then if test -f SATISFIED; then \
 		echo "Testing for Lisp shadows ..."; \
-		./${PROGNAME} -batch -vanilla -f list-load-path-shadows; fi; \
+		$(DUMPENV) ./${PROGNAME} -batch -vanilla -f list-load-path-shadows; fi; \
 		$(RM) SATISFIED; exit 0; fi; \
 	if test -f SATISFIED; then $(RM) SATISFIED; exit 1; fi; \
 	$(RECURSIVE_MAKE) $@;
@@ -344,7 +344,7 @@
 	@$(RM) ${PROGNAME} && touch SATISFIED
 	-$(DUMPENV) ./temacs -batch -l ${srcdir}/../lisp/loadup.el dump
 	@if test -f ${PROGNAME};        then if test -f SATISFIED; then \
-		./${PROGNAME} -batch -vanilla -f list-load-path-shadows; fi; \
+		$(DUMPENV) ./${PROGNAME} -batch -vanilla -f list-load-path-shadows; fi; \
 		$(RM) SATISFIED; exit 0; fi; \
 	if test -f SATISFIED; then $(RM) SATISFIED; exit 1; fi;
 
