--- xemacs-20.4/src/Makefile.in.in.orig	Mon Feb  2 06:30:28 1998
+++ xemacs-20.4/src/Makefile.in.in	Wed Mar  4 19:20:33 1998
@@ -385,8 +385,29 @@
  ${start_flags} ${ldflags}\
  -o $@ ${start_files} ${objs} ${otherobjs} ${LIBES}
 
+temacs_link_args_static_i386=\
+ ${start_flags} ${ldflags} -Wl,-Bstatic\
+ -o $@ ${start_files} ${objs} ${otherobjs} ${LIBES}
+
+temacs_link_args_static_sparc=\
+ ${start_flags} ${ldflags} -Wl,-Bstatic -Wl,-noinhibit-exec\
+ -o $@ ${start_files} ${objs} ${otherobjs} ${LIBES}
+
+temacs_link_args_static_alpha=\
+ ${start_flags} ${ldflags} -Wl,-Bstatic\
+ -o $@ ${start_files} ${objs} ${otherobjs} ${LIBES}
+
 temacs: $(temacs_deps)
 	$(LD) $(temacs_link_args)
+
+temacs-static-i386: $(temacs_deps)
+	$(LD) $(temacs_link_args_static_i386)
+
+temacs-static-sparc: $(temacs_deps)
+	$(LD) $(temacs_link_args_static_sparc)
+
+temacs-static-alpha: $(temacs_deps)
+	$(LD) $(temacs_link_args_static_alpha)
 
 .PHONY : run-temacs
 
